src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -11893,6 +11893,17 @@ public async Task ParameterAvailableInDelegateParameterAttributeNameof()
             await VerifyItemExistsAsync(MakeMarkup(source, languageVersion: "10"), "parameter", skipSpeculation: true);
         }
 
+        [Fact, WorkItem(64585, "https://github.com/dotnet/roslyn/issues/64585")]
+        public async Task AfterRequired()
+        {
+            var source = @"
+class C
+{
+    required $$
+}";
+            await VerifyAnyItemExistsAsync(source);
```

Comment:
Should we validate that keywords appear as well (like 'int')? Or do those tests already exist?

Reply:
That was fixed by this PR as well, so there is no existing test there. Currently there are no completions after `required` at all, this PR starts treating `required` as modifier, which enables _a lot of_ completions: symbols, symbol keywords (`int`, `long` etc.), snippets.